MuonAnalysis/MuonAssociators/src/PropagateToMuonSetup.cc

```diff
@@ -0,0 +1,78 @@
+#include "MuonAnalysis/MuonAssociators/interface/PropagateToMuonSetup.h"
+
+#include <cmath>
+
+#include "DataFormats/RecoCandidate/interface/RecoCandidate.h"
```

Comment:
Is this needed?
(The same for several other headers included here below)

Reply:
No, removed (same goes for a few others)